hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoCreateValidator.java

```diff
@@ -77,16 +79,26 @@ public void validateKeyAliasAndEvmAddressCombinations(
         }
     }
 
+    private void validateKey(
+            @NonNull final CryptoCreateTransactionBody op, @NonNull final AttributeValidator attributeValidator) {
+        final var key = op.key();
+        if (isEmpty(key)) {
+            throw new HandleException(KEY_REQUIRED);
+        }
+        if (!isValid(key)) {
+            throw new HandleException(INVALID_ADMIN_KEY);
+        }
+        attributeValidator.validateKey(op.keyOrThrow());
```

Comment:
We already have the key as `key`, so why not `attributeValidator.validateKey(key);`?

Reply:
ah yes .. fixed.